src/js/components/Drop/stories/Progressive.js

```diff
@@ -24,46 +22,47 @@ const ProgressiveDrop = () => {
   };
 
   return (
-    <Grommet theme={grommet} full>
```

Comment:
+1

Reply:
I think we could either have the `full` prop or add padding to the box container so the contents are more centered.